config/config.go

```diff
@@ -888,6 +889,7 @@ func (a AlertmanagerConfigs) ToMap() map[string]*AlertmanagerConfig {
 
 // AlertmanagerAPIVersion represents a version of the
 // github.com/prometheus/alertmanager/api, e.g. 'v1' or 'v2'.
+// V1 is current deprecated and no longer supported.
```

Comment:
```suggestion
// 'v1' is no longer supported.
```

To avoid any confusion with "deprecated"

Reply:
Going by https://github.com/prometheus/alertmanager/issues/2469#issue-796945234
Should we add here that this consequently required at least alertmanager v0.16?

Reply:
we can mention that in https://github.com/search?q=repo%3Aprometheus%2Fprometheus%20%22The%20api%20version%20of%20Alertmanager%22&type=code as well to make it more visible to users.